scss/bitstyles/base/input-checkbox/_settings.scss

```diff
@@ -6,7 +6,7 @@
 // Base styles ////////////////////////////////////////
 $border-radius: size.get('xxs') !default;
 $gap: size.get('xs') !default;
-$size: size.get('m') !default;
+$size: size.get('l') !default;
```

Comment:
Was this a bug?

Reply:
Something I found along the way, which should be in its own PR (coming soon…)